### PR TITLE
Upgrade gatsby to v2

### PR DESCRIPTION
**Pull request by mattcleary55**

### CODE REVIEW

File: src/templates/flyUnlimited.js [hunk 1]
```diff
@@ -37,141 +39,143 @@ class FlyUnlimitedPageTemplate extends Component {
...
+                    <img src={FUnBadge} css={style.heroImage} />
+                  </Flex>
+                )}
+                textColor={colors.white}
+                header={hero.header}
+                description={hero.description}
+                buttons={hero.buttons}
+                features={hero.features}
+                smallPrint={
+                  '*Maximum £75 discount off first month. [T&C\'s](https://help.flockcover.com/legal/free-month-fly-unlimited-tcs) apply.'
```
Comment (jbreckmckye) [inline src/templates/flyUnlimited.js]: At some point we might think about injecting text from Flock-Lang.

File: src/components/RenewalBanner.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import R from 'ramda'
-import {css, injectGlobal, cx} from 'react-emotion'
+import * as R from 'ramda'
+import {css} from '@emotion/core'
+import {injectGlobal} from 'emotion'
```
[the remark this answers is not in the document]
Comment (jbreckmckye) [reply src/components/RenewalBanner.js]: Not sure what will happen if we skip this step, as the site seems to work just fine anyway.

File: src/templates/flyUnlimited.js (hunk 1 above)
Comment (mattcleary55) [reply src/templates/flyUnlimited.js]: Definitely, we'll get there :)

File: src/components/Layout/Layout.js [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+                      textAlign: 'center',
+                      lineHeight: '2.5vw'
+                    }}
+                    location="top"
+                  >
+                    <span>
+                      This website uses cookies to enhance the user experience.
+                      &nbsp;{' '}
+                      <a
+                        css="cookiesLinkSmall"
```
[the remark this answers is not in the document]
Comment (mattcleary55) [reply src/components/Layout/Layout.js]: Classic `find/replace all` error :) Thanks!

**Pull request merged**